main/main.cpp

```diff
@@ -2619,6 +2619,20 @@ bool Main::start() {
 
 	OS::get_singleton()->set_main_loop(main_loop);
 
+	uint64_t minimum_time_msec = GLOBAL_DEF("application/boot_splash/minimum_display_time_msec", 0);
```

Comment:
Seems like this is not picked up by doctool as the CI wants to remove it from the XML. It might need to be defined somewhere else. (CC @KoBeWi who reworks this in #61549).

Reply:
This should be moved way up before the doctool code (around line 2120).

Reply:
I meant moving only definition, but that works too I guess.

Reply:
I think I better fix it.